stats.py

```diff
@@ -437,6 +464,17 @@ def count_solutions(df, solutions=True):
 
     return df_
 
+def handle_files(files):
```

Comment:
same here

Reply:
why this docstring empty? this functions seems so hacky... why are you counting the `/` anyway?

Reply:
double "/" for "ProblemXXX/Lang/solution
one "/" is for readme and hashes so its ignored
none of them means a file in /ProjectEuler directory

Reply:
ok